pkg/scheduler/plugins/nodeorder/nodeorder.go

```diff
@@ -130,7 +130,7 @@ func calculateWeight(args framework.Arguments) priorityWeight {
 
 func (pp *nodeOrderPlugin) OnSessionOpen(ssn *framework.Session) {
 	weight := calculateWeight(pp.pluginArguments)
-	pl := util.NewPodLister(ssn)
+	pl := util.NewPodListerFromNode(ssn)
```

Comment:
Will [PR 1243](https://github.com/volcano-sh/volcano/pull/1243) resolves this issue? l think in that pr `nodemap` contains all status' pods

Reply:
nodeOrderFn function score doesn't base  on  nodemap   but base on handle [handle := k8s.NewFrameworkHandle(pods, nodeSlice) ], and  pods info in handle struct data is create by "pl := util.NewPodLister(ssn)"

Reply:
yes, but in that PR it modifies the function `NewFrameworkHandle` so that it doesn't depend on `pods` param any more, then the snapshot will contain all the pods.